README.md

```diff
@@ -18,7 +19,7 @@ This project aims to :
 
 # Repository
 
-The repository uses GitHub Pages to expose the Helm charts at (http://snowdrop.github.io/helm)[http://snowdrop.github.io/helm]. To use it, you need to execute:
+The repository uses GitHub Pages to expose the Helm charts at [http://snowdrop.github.io/helm](http://snowdrop.github.io/helm). To use it, you need to execute:
```

Comment:
`The repository uses GitHub Pages to expose the Helm charts at []()` ==> `This repository uses GitHub Pages to publish the Helm charts index at this address: [....]()`

` To use it ...` ==> `To use it locally ....`

Reply:
Changed